vaxrank/core_logic.py

```diff
@@ -49,6 +49,16 @@ def vaccine_peptides_for_variant(
         variant=variant,
         protein_sequence=isovar_protein_sequences[0])
 
+    min_expected_length = mhc_predictor.min_peptide_length
+    def_expected_length = mhc_predictor.default_peptide_lengths
+    if def_expected_length is not None and len(def_expected_length) > 0:
```

Comment:
do we want the default peptide lengths to override the min peptide length setting in this case? that seems a bit counterintuitive. it might be cleaner to catch ValueErrors from the `predict_subsequences` call in `epitope_prediction.predict_epitopes()` and log those errors before returning early